ext/iperf3/iperf_api.c

```diff
@@ -1642,19 +1642,19 @@ int iperf_parse_arguments(struct iperf_test *test, int argc, char **argv)
     if (!test->curr_mdm_traces) {
```

Comment:
In this file `CONFIG_AT_CMD` is still used in several places.

Reply:
Thank you, I have should have removed all of those now both in `iperf3` and `curl`.